packages/grafana-ui/src/components/AutoSaveField/AutoSaveField.mdx

```diff
@@ -0,0 +1,277 @@
+import { Props, Preview } from '@storybook/addon-docs/blocks';
+import { AutoSaveField } from './AutoSaveField';
+import { Basic } from './AutoSaveField.story.tsx';
+
+# AutoSaveField
+
+Used for text input that saves automatically its content. For an array of data or tree-structured data, consider using `Select` or `Cascader` respectively.
```

Comment:
```suggestion
Used for form inputs that should save its content automatically. 
```

Also do we want to add what is at the top of the autosave doc as a sort of "warning" to use this sparingly?

Reply:
If we want a "warning" I think this is the best place to show it.